readme_tool/figshare.py

```diff
@@ -0,0 +1,87 @@
+from typing import Union
+import requests
+
+from fastapi import FastAPI, HTTPException
+# from pydantic import BaseModel
+
+api_version = 'v1'
+
+app = FastAPI()
+
+
+def figshare_metadata_readme(figshare_dict: dict) -> dict:
+    """
+    Function to provide shortened dict for README metadata
+
+    :param figshare_dict: Figshare API response
+
+    :return: README metadata based on Figshare response
+    """
+    single_str_citation = figshare_dict['citation']
+
+    # Handle period in author list.  Assume no period in dataset title
```

Comment:
In the future, it would be great to have an example of the raw string in a comment.

Reply:
This might be better embedded in the Swagger UI as examples. I strip the code from LD-Cool-P.

Reply:
I created #5